src/averageAges.js

```diff
@@ -15,7 +27,17 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
+  const men = people.filter(person => findMale(person));
+  let menAges = [];
+
+  if (century !== undefined) {
+    menAges = men.filter(person =>
+      Math.ceil(person.died / 100) === century).map(findAge);
+  } else {
+    menAges = men.map(findAge);
+  }
```

Comment:
use ternary to assign either of two options:
```suggestion
  let menAges = century !== undefined
    ? men.filter(person =>
        Math.ceil(person.died / 100) === century).map(findAge)
    : men.map(findAge);
```